### PR TITLE
additional nb60_medical_imaging updates

### PR DESCRIPTION
**Pull request by asvcode**
Additional updates to notebook 60

### CODE REVIEW

File: nbs/60_medical.imaging.ipynb [hunk 1]
```diff
@@ -415,7 +415,7 @@
     {
```
Comment (jph00) [inline nbs/60_medical.imaging.ipynb:415]: <p>All docstrings should be in double quotes: " . Not in tripled double quotes: """</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2772/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='97'/>

Comment (jph00) [inline nbs/60_medical.imaging.ipynb:415]: <p>I'd remove this output - it's a lot of space that doesn't say much</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2772/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='99'/>

**Review by jph00 — CHANGES_REQUESTED** [review]
Looks good! Just some very minor requests.

File: nbs/60_medical.imaging.ipynb (hunk 1 above)
Comment (asvcode) [reply nbs/60_medical.imaging.ipynb:415]: <p>lol don't know what I was thinking there - all corrected</p>

Comment (asvcode) [reply nbs/60_medical.imaging.ipynb:415]: <p>got it, removed</p>

**Comment by asvcode** [comment]
Thanks!  Having issues submitting the corrections hence creating a new PR #2780 with the updates and closing this one

**Pull request closed without merging**